files/splunk_hec_collect_api_events.rb

```diff
@@ -123,7 +127,7 @@ def main(confdir, modulepaths)
 
     # get the events using the limit
     events = events_client.get_events(service: service, limit: new_jobs)
-    result = process_response(events.items, events.total, settings, store_file, 'puppet:activity', splunk_client)
+    result = process_response(events.items, events.total, settings, store_file, source_type, splunk_client)
```

Comment:
The readme will need updates to lines 210-211 216-218. There might be some more. 
We will need to update the TA report viewer

Reply:
README is updated. I'll talk to @gsparks about updating the report viewer.